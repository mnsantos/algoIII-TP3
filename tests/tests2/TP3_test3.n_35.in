35 197
1 10
2 10
2 17
1 2
1 8
5 6
1 7
2 8
1 3
5 7
5 14
3 7
1 4
3 13
9 10
2 9
1 16
4 9
1 17
4 6
1 11
1 18
5 8
7 15
1 9
7 14
32 35
32 33
31 33
31 32
8 31
8 30
8 29
8 27
8 22
8 21
8 20
7 35
7 34
7 27
7 25
7 22
7 19
7 17
6 29
6 25
6 24
6 23
6 20
6 17
5 33
5 31
5 30
5 28
5 23
5 22
5 20
5 18
4 34
4 26
4 25
4 24
3 35
3 34
3 32
3 27
3 26
3 24
3 21
1 26
1 25
1 24
1 23
1 22
1 21
1 20
1 19
1 27
1 28
1 29
1 30
1 31
1 32
1 33
1 34
1 35
2 20
2 21
2 22
2 23
2 27
2 32
3 19
3 20
8 32
8 33
9 14
9 17
9 18
9 24
9 28
9 31
9 35
10 17
10 27
10 29
10 33
10 34
10 35
11 20
11 22
11 25
11 34
12 15
12 16
12 30
13 17
13 20
13 21
13 28
13 29
13 35
14 15
14 16
14 19
14 21
14 29
14 33
15 17
15 19
15 22
15 24
15 25
15 34
16 18
16 20
16 23
16 27
16 30
16 32
16 34
17 19
17 20
17 22
17 25
17 29
17 35
18 20
18 21
18 27
18 32
18 34
18 35
19 26
19 28
19 29
19 33
20 28
21 22
21 24
21 27
22 23
22 25
22 26
22 35
23 24
23 26
24 25
24 29
25 26
25 28
25 32
25 35
26 27
26 31
26 32
26 33
26 34
27 29
27 31
28 29
28 30
28 31
28 32
29 30
30 34
30 35
1 15
1 13
10 13
9 12
2 14
1 6
1 12
1 14
1 5
4 16
0
