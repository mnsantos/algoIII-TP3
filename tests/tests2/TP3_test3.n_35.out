68 6 1 7 17 22 25 35
